packages/core/utils/src/installPackage.js

```diff
@@ -47,10 +58,14 @@ async function install(modules, filepath, options = {}) {
   }
 }
 
-async function installPeerDependencies(filepath, name, options) {
+async function installPeerDependencies(
+  filepath: FilePath,
+  name: string,
+  options
+) {
   let basedir = path.dirname(filepath);
-  const resolved = await resolve(name, {basedir});
-  const pkg = await config.loadConfig(resolved, ['package.json']).config;
+  const [resolved] = await resolve(name, {basedir});
```

Comment:
`util.promisify` looks like it only resolves with the first non-error value in the callback, whereas ours will pack non-error values into an array. Some places used `util.promisify`, and some used our own `promisify`. This unifies them, though in this case the resolved path needs to be unpacked.